diff --git a/lib/esm/fastembed.js b/lib/esm/fastembed.js
--- a/lib/esm/fastembed.js
+++ b/lib/esm/fastembed.js
@@ -4,7 +4,7 @@ import https from "https";
 import * as ort from "onnxruntime-node";
 import path from "path";
 import Progress from "progress";
-import tar from "tar";
+import * as tar from "tar";
 import { downloadFileToCacheDir } from "@huggingface/hub";
 export var ExecutionProvider;
 (function (ExecutionProvider) {
